lib/decode/decode.go

```diff
@@ -73,14 +73,19 @@ func translationsForType(to reflect.Type) map[string]string {
 	translations := map[string]string{}
 	for i := 0; i < to.NumField(); i++ {
 		field := to.Field(i)
+		canonKey := strings.ToLower(canonicalFieldKey(field))
```

Comment:
@dnephin After some recent changes to manually translate the protobuf config to the internal config this may not be necessary. Do you think its worthwhile to keep this behavior in that case?

Reply:
It seems fine to me to keep it with `ToLower`